core/constants.py

```diff
@@ -48,3 +48,17 @@
 YEAR_CHOICES = []
 for year in range(2013, (datetime.now().year + 1)):
     YEAR_CHOICES.append((year, year))
+
+NEWLY_REPORTED = "Newly reported"
+INTERNAL_ASSESSMENT = "Internel assessment"
+REQUIRES_REPAIR = "Requires repair"
```

Comment:
Please view this story to see how state transitions should be implemented https://www.pivotaltracker.com/story/show/164099111